api/app/src/test/kotlin/packit/integration/MetricsTest.kt

```diff
@@ -0,0 +1,33 @@
+package packit.integration
+
+import org.junit.jupiter.api.Test
+import org.springframework.boot.test.autoconfigure.actuate.observability.AutoConfigureObservability
+import org.springframework.boot.test.web.client.TestRestTemplate
+import org.springframework.boot.test.web.server.LocalManagementPort
+import org.springframework.boot.web.client.RestTemplateBuilder
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+@AutoConfigureObservability
```

Comment:
Just curious - why did you need to add this?

Reply:
Spring disables metrics in tests by default

https://stackoverflow.com/a/65560631
https://github.com/spring-projects/spring-boot/pull/21658